cs/v1/CS_v1.0.0.yaml

```diff
@@ -663,104 +739,61 @@ components:
         - carrierExportVoyageNumber
         - timestamps
       properties:
-        carrierServiceName:
```

Comment:
`carrierServiceName` plus a lot of other fields are mentioned as required. Please have a look at the required fields for this object